packages/react/src/internal/components/ContentBlock/ContentBlock.js

```diff
@@ -65,7 +65,14 @@ const ContentBlock = ({
 
   return (
     <div data-autoid={`${stablePrefix}--content-block`} className={classnames}>
-      {title}
+      {aside && aside.items
+        ? _layoutWrap(
+            <>
+              {title}
+              <div></div>
+            </>
+          )
+        : title}
```

Comment:
hmm wondering why this is being added back? It causes the title to go off grid with aside elements. Otherwise looks good!!!
<img width="644" alt="Screen Shot 2021-06-02 at 7 20 45 AM" src="https://user-images.githubusercontent.com/20210594/120497311-24251f00-c373-11eb-9a02-98c01a100841.png">

Reply:
I removed what I added back since it's more noticeable with that code. I've looked a bit more into this, and I think this is affecting all Content Blocks whenever there's the LinkList open. I'll be opening a new issue capturing this layout problem if there's not one already so it can be addressed properly.

Reply:
update: after merging latest changes, I was able to remove the left padding from the content and everything renders as expected. No need to create an issue as it's been already fixed.